packages/ui-demo/App.tsx

```diff
@@ -20,26 +21,18 @@ for (const story of stories) {
     allStories[storyName] = story.stories[storyName];
   }
 }
-console.info(
-  "ALL",
-  Object.keys(allStories),
-  stories.map((s) => s.title)
-);
-
-const renderStory = (story: Story) => {
-  return (
-    <View>
-      <Text>{story.title}</Text>
-      {Object.keys(story.stories).map((key) => {
-        const Story = story.stories[key];
-        return <Story key={key} />;
-      })}
-    </View>
-  );
-};
 
 const App = () => {
   const [currentStory, setStory] = useState<string | null>(null);
+
+  useEffect(() => {
+    AsyncStorage.getItem("story").then((story) => {
+      if (story) {
+        setStory(story);
+      }
+    });
+  }, []);
```

Comment:
I'm coming around to hooks, but I do not like this pattern. But such is life heh.